lib/zht/zht-main-page.dart

```diff
@@ -6,55 +6,136 @@ import 'package:OutOfBounds/zht/pages/ActualPosition.dart';
 import 'package:OutOfBounds/zht/pages/SettingsPage.dart';
 import 'package:provider/provider.dart';
 
+const WIDTH = 106.5;
+const HEIGHT = 50.0;
+const TEXT_SIZE = 7.5;
+const ICON_SIZE= 12.0;
 class ZhtMainPage extends StatefulWidget {
   @override
   State<StatefulWidget> createState() {
     return MyAppState();
   }
 }
-
 class MyAppState extends State<ZhtMainPage> {
   int whoSelected = 0;
   final itemsChoice = [
     StartPosition(),
     ActualPosition(),
     SettingsPage(),
   ];
+  Alignment Aligne = Alignment.bottomLeft;
+  Color new_colors_starting =  Color.fromARGB(255, 128, 128, 128);
```

Comment:
codeMagic fail. use lower camel case for non constant fields => newColorsStarting